client/components/select-dropdown/style.scss

```diff
@@ -120,6 +120,15 @@ $compact-header-height: 35;
 		right: 40px;
 		top: #{ ( $header-height - 18 - 2 ) / 2 }px;
 	}
+
+	.gridicon {
+		margin-right: #{ ( $side-margin / 4 ) }px;
```

Comment:
This feels a little tight. I changed it from `4px` to `8px` in Chrome's dev tools when checking it out and it felt much better.